dist/common/scripts/scylla_kernel_check

```diff
@@ -42,7 +39,23 @@ if __name__ == '__main__':
     run('umount /var/tmp/mnt', shell=True, check=True)
     shutil.rmtree('/var/tmp/mnt')
     os.remove('/var/tmp/kernel-check.img')
-    if ret == 0:
+    return (ret == 0)
+
+if __name__ == '__main__':
+    if os.getuid() > 0:
+        print('Requires root permission.')
+        sys.exit(1)
+
+    result = True
+    if is_redhat_variant() and distro.version() == '7':
+        arch = platform.machine()
+        current_kernel = version.parse(platform.release().strip(f'.{arch}'))
+        acceptable_kernel = version.parse('3.10.0-514.el7')
+        if current_kernel < acceptable_kernel:
+            result = False
+    if result:
+        result = fs_check()
+    if result:
```

Comment:
See https://github.com/scylladb/scylla/issues/7675#issuecomment-734146850. Turns out it's not related to aio at all.

I think this check comes too late, at this point scylla.rpm is already installed and we will get the errors from systemd/kernel.

Maybe we can drop the conflicts and not add a new check (like the first version of the patch). CentOS 7.3 was released on Dec 2016, 4 years ago, and if someone did not upgrade yet, they will have an error.

Reply:
@avikivity reverted to first version, changed the commit message since we don't do version check on setup script.